compiler/cli/bin/kotlinc.bat

```diff
@@ -28,6 +28,10 @@ set _arg=%~1
 if "%_arg%" == "" goto loopend
 
 if "%_arg:~0,2%"=="-J" (
+  if "%_arg:~2%"=="" (
+    echo error: empty -J argument
+    goto error
```

Comment:
The `exit /b %ERRORLEVEL%` won't work here, so I add a goto `clause` here (otherwise it requires ERRORLEVEL to be a `DelayedExpansion` which looks a little more complicated to me).